jboss/container/eap/openshift/modules/module.yaml

```diff
@@ -8,8 +8,8 @@ execute:
 
 artifacts:
 - name: txn-recovery-marker-jdbc-common
-  target: txn-recovery-marker-jdbc-common-1.1.4.Final-redhat-00001.jar
+  target: txn-1.1.4.Final-redhat-00001.jar
   md5: 305aa706018b1089e5b82528b601541f
 - name: txn-recovery-marker-jdbc-hibernate5
-  target: txn-recovery-marker-jdbc-hibernate5-1.1.4.Final-redhat-00001.jar
+  target: txn-1.1.4.Final-redhat-00001.jar
```

Comment:
this is the same as the one above, so it'll overwite it. You'll need to keep the -common and -jdbc perhaps and also update the configure script to match.

This should just be tested in a branch that CPaaS uses first, as we don't know if this is actually the reason